lib/phoenix/live_dashboard/pages/processes_page.ex

```diff
@@ -38,7 +39,7 @@ defmodule Phoenix.LiveDashboard.ProcessesPage do
     %{search: search, sort_by: sort_by, sort_dir: sort_dir, limit: limit} = params
 
     {processes, count, state} =
-      SystemInfo.fetch_processes(node, search, sort_by, sort_dir, limit, state)
+      SystemInfo.fetch_processes(node, params.filter, search, sort_by, sort_dir, limit, state)
```

Comment:
As mention in the previous discussion, we will not pass the filter to the node. Instead, the node will return the filter.  You can think that this function call is now return something like:

    {active_filter, available_filters, processes, count, state} =

:)

Reply:
Agreed, and I think I'm almost there :-)

Reply:
Awesome! I think we will end up with less code and conditionals than currently. :) 🤞

Reply:
Okay, I think it's in decent shape now. I tested it with a single node (with or without filter configured), with 2-node cluster (no filters/same filters/different filters on both, filter on one, no filter on another...). Seems to work in conjunction with a text search and limits.
Also, the filter injection into the table component goes through row_fetcher. I couldn't find a better way without rewriting too much code there. There could well be a better way :-)